README.md

```diff
@@ -102,25 +102,27 @@ jobs:
 
 See the example project [component-tests](examples/component-tests/) and the [example-component-test.yml](.github/workflows/example-component-test.yml) workflow for more details.
 
-### Explicit version
+### Action version
 
 **Best practice:**
 
-Our examples specify the tag of the action to use listing only the major version `@v5`
+Our examples specify using branch [v5](https://github.com/cypress-io/github-action/tree/v5) which is the action's latest major version:
 
 ```yml
 - name: Cypress run
   uses: cypress-io/github-action@v5
 ```
 
-When using `cypress-io/github-action@v5` from your workflow file, you automatically will be using the latest [tagged version from this repository](https://github.com/cypress-io/github-action/tags). If you want to precisely control the version of this module, use the full tag version, for example:
+When using `cypress-io/github-action@v5` from your workflow file, you will automatically use the latest [tag](https://github.com/cypress-io/github-action/tags) from branch [v5](https://github.com/cypress-io/github-action/tree/v5).
+
+Alternatively, to mitigate unforeseen breaks, bind to a specific [tag](https://github.com/cypress-io/github-action/tags), for example:
 
 ```yml
 - name: Cypress run
   uses: cypress-io/github-action@v5.1.0
 ```
 
-By using the full version tag, you will avoid accidentally using a newer version of the action.
+The changes associated with each tag are shown under GitHub's [releases](https://github.com/cypress-io/github-action/releases) list. Refer also to the [Changelog](#changelog) for an overview of major changes.
```

Comment:
```suggestion
The changes associated with each tag are shown under GitHub's [releases](https://github.com/cypress-io/github-action/releases) list. Refer also to the [Changelog](#changelog) for an overview of release changes.
```
are we only documented breaking changes?

Reply:
@emilyrohrbough

Thanks for your review!

The repository uses a default setup of [cycjimmy/semantic-release-action](https://github.com/cycjimmy/semantic-release-action) to produce releases. This is documented on the standard GitHub [releases](https://github.com/cypress-io/github-action/releases) page. Also there is another open PR https://github.com/cypress-io/github-action/pull/853 which goes into more detail about the current release process, which you might find helpful.

There is no automatic Changelog produced. There is a pending draft PR https://github.com/cypress-io/github-action/pull/752 to activate the production of a changelog, which is waiting for @jaffrepaul to have time on improving the release workflow. To begin with, @jaffrepaul was editing the individual release entries on the [releases](https://github.com/cypress-io/github-action/releases) page to include more detail, including commits which were not triggering a release, but lately he has stopped doing this. Presumably due to lack of time.

Because there was no overview of any releases, let alone major releases, and the existing migration guide was unhelpful, I added an overview of major releases in PR https://github.com/cypress-io/github-action/pull/769. This is just a short table included in the [README: Changelog](https://github.com/cypress-io/github-action/blob/master/README.md#changelog).

So, in summary, the original text is correct and your suggested change would be inaccurate for this repository.

I'm happy to work together if there is any work done to improve the release workflow, however I do appreciate your heavy workload and I'm not expecting anything soon on this.

Reply:
@MikeMcC399 Wow! Thank you for the full context!! I am just ramping up with this project and where it is at. @jaffrepaul Recently shifted teams and focus at Cypress and likely won't be actively/frequently contributing to this project due to his shift in focus.

Reply:
@emilyrohrbough

> Wow! Thank you for the full context!! I am just ramping up with this project and where it is at. @jaffrepaul Recently shifted teams and focus at Cypress and likely won't be actively/frequently contributing to this project due to his shift in focus.

You're very welcome! Thanks also for merging!

I look forward to working with you in future in that case! Thanks for the heads-up about @jaffrepaul. I've been in touch with him via Discord and lately I haven't been hearing much from him on that channel.

Reply:
I'm looking forward to working with you as well! You've made a large, GOOD, impression on our team and we have been super excited about all of the efforts and contributions you have made to help us get things on track and up-to-date! We are super super appreciative!

Reply:
@emilyrohrbough 
Thanks very much! That's nice to hear 🙂 ! The action itself is in good shape. It's now more a question of getting documentation and tutorials, etc. up to date.